desktop/src/test/java/com/frostwire/tests/One337xSearchPerformerTest.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Created by Angel Leon (@gubatron), Alden Torres (aldenml), Himanshu Sharma (HimanshuSharma789)
+ * Copyright (c) 2011-2018, FrostWire(R). All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.frostwire.tests;
+
+import com.frostwire.regex.Matcher;
+import com.frostwire.regex.Pattern;
+import com.frostwire.search.SearchMatcher;
+import com.frostwire.search.one337x.One337xSearchPerformer;
+import com.frostwire.search.one337x.One337xSearchResult;
+import com.frostwire.util.ThreadPool;
+import com.frostwire.util.http.HttpClient;
+import com.frostwire.util.http.OKHTTPClient;
+
+import java.util.concurrent.LinkedBlockingQueue;
+
+public final class One337xSearchPerformerTest {
+    public static void main(String[] args) throws Throwable {
+        String TEST_SEARCH_TERM = "avengers";
```

Comment:
please remove this search term as it induces copyright infringement. Use something like "foobar"